### PR TITLE
Tweak: Add labels to font weight numeric values [ED-7670]

### LINKED ISSUE #18761: 🚀 Display the name of each weight in the Custom Font Addition section
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When adding a font in elementor, [only the font weight is displayed](https://prnt.sc/c9h3sYRgYeR7). Therefore, most users do not know, for example, what the name of the  400 is.
So, it would be great if you write the name of each font against its weight : https://prnt.sc/kq5xMPAwWVur
This is a very simple but important feature. Please add it.

### Describe the solution you'd like

https://prnt.sc/_O1l510M-CrQ

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by rami-elementor**
## PR Checklist
- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe: i18n

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add labels to font weight numeric values

## Description
An explanation of what is done in this PR

* Tweak: Add labels to font weight numeric values

Fixes #18761

### CODE REVIEW

File: includes/controls/groups/typography.php [hunk 1]
```diff
@@ -133,15 +133,15 @@ protected function init_fields() {
...
+				'300' => esc_html_x( '300 (Light)', 'Typography Control', 'elementor' ),
+				'400' => esc_html_x( '400 (Normal)', 'Typography Control', 'elementor' ),
+				'500' => esc_html_x( '500 (Medium)', 'Typography Control', 'elementor' ),
+				'600' => esc_html_x( '600 (Semi Bold)', 'Typography Control', 'elementor' ),
+				'700' => esc_html_x( '700 (Bold)', 'Typography Control', 'elementor' ),
+				'800' => esc_html_x( '800 (Extra Bold)', 'Typography Control', 'elementor' ),
+				'900' => esc_html_x( '900 (Black)', 'Typography Control', 'elementor' ),
 				'' => esc_html__( 'Default', 'elementor' ),
 				'normal' => esc_html__( 'Normal', 'elementor' ),
 				'bold' => esc_html__( 'Bold', 'elementor' ),
```
Comment (StyleShit) [inline includes/controls/groups/typography.php:147]: shouldn't those ones also have `esc_html_x`?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: includes/controls/groups/typography.php (hunk 1 above)
Comment (rami-elementor) [reply includes/controls/groups/typography.php:147]: Yes they should, but if we replace `esc_html__()` with `esc_html_x()`, we will have to translate those string all over again in [translate.WordPress.org](https://translate.WordPress.org/). A translation string with context is a different translation string then a translation string without context. For now, let's leave as it is. I will create a separate PR for i18n issue.

Comment (arielk) [reply includes/controls/groups/typography.php:147]: @rami-elementor I think you can change it in this PR, its only 3 strings.

Comment (rami-elementor) [reply includes/controls/groups/typography.php]: @arielk @StyleShit Done.

File: includes/controls/groups/typography.php [hunk 2]
```diff
@@ -133,18 +133,18 @@ protected function init_fields() {
...
-				'bold' => esc_html__( 'Bold', 'elementor' ),
+				'100' => esc_html_x( '100 (Thin)', 'Typography Control', 'elementor' ),
+				'200' => esc_html_x( '200 (Extra Light)', 'Typography Control', 'elementor' ),
+				'300' => esc_html_x( '300 (Light)', 'Typography Control', 'elementor' ),
+				'400' => esc_html_x( '400 (Normal)', 'Typography Control', 'elementor' ),
+				'500' => esc_html_x( '500 (Medium)', 'Typography Control', 'elementor' ),
+				'600' => esc_html_x( '600 (Semi Bold)', 'Typography Control', 'elementor' ),
+				'700' => esc_html_x( '700 (Bold)', 'Typography Control', 'elementor' ),
+				'800' => esc_html_x( '800 (Extra Bold)', 'Typography Control', 'elementor' ),
+				'900' => esc_html_x( '900 (Black)', 'Typography Control', 'elementor' ),
```
Comment (KingYes) [inline includes/controls/groups/typography.php:144]:
```suggestion
				'100' => '100 ' . esc_html_x( '(Thin)', 'Typography Control', 'elementor' ),
				'200' => '200 ' . esc_html_x( '(Extra Light)', 'Typography Control', 'elementor' ),
				'300' => '300 ' . esc_html_x( '(Light)', 'Typography Control', 'elementor' ),
				'400' => '400 ' . esc_html_x( '(Normal)', 'Typography Control', 'elementor' ),
				'500' => '500 ' . esc_html_x( '(Medium)', 'Typography Control', 'elementor' ),
				'600' => '600 ' . esc_html_x( '(Semi Bold)', 'Typography Control', 'elementor' ),
				'700' => '700 ' . esc_html_x( '(Bold)', 'Typography Control', 'elementor' ),
				'800' => '800 ' . esc_html_x( '(Extra Bold)', 'Typography Control', 'elementor' ),
				'900' => '900 ' . esc_html_x( '(Black)', 'Typography Control', 'elementor' ),
```

**Review by KingYes — CHANGES_REQUESTED** [review]

**Comment by nicholaszein** [comment]
@rami-elementor, @KingYes please consider this comment: https://github.com/orgs/elementor/discussions/19854#discussioncomment-3708688

**Comment by arielk** [comment]
@nicholaszein You answered well, I agree that it is not ideal but once we have these variables it is very difficult to delete them.

**Pull request merged**